gke-deploy/deployer/deployer.go

```diff
@@ -47,7 +47,7 @@ type Deployer struct {
 }
 
 // Prepare handles preparing deployment.
-func (d *Deployer) Prepare(ctx context.Context, images []name.Reference, appName, appVersion, config, output, namespace string, labels map[string]string) error {
+func (d *Deployer) Prepare(ctx context.Context, images []name.Reference, appName, appVersion, config, output, namespace string, labels map[string]string, exposePort int) error {
```

Comment:
As a general comment, these args could be encapsulated into a struct, maybe even the `options` struct they originally come from, exported and moved into this package. Not urgent, just something to consider.

Reply:
The parameter list is getting pretty long for this function. I'll consider doing that the next time a new arg is added.